packages/netlify-cms-core/src/integrations/providers/algolia/implementation.js

```diff
@@ -129,6 +129,37 @@ export default class Algolia {
     }
   }
 
+  async listAllEntries(collection) {
```

Comment:
We can re-visit and UI server side sorting. For now I'm just getting all entries and sorting client side

Reply:
Agreed.